C/CoverM/build_tarballs.jl

```diff
@@ -3,13 +3,13 @@
 using BinaryBuilder, BinaryBuilderBase
 
 name = "CoverM"
-version = v"0.6.1"
+version = v"0.7.0"
 
 # Collection of sources required to complete build
 sources = [
     GitSource(
-        "https://github.com/wwood/CoverM",
-        "0056336d4b245064a23e5b40197b29607fb1fe56"),
+        "https://github.com/jakobnissen/CoverM",
```

Comment:
Why a fork, instead of applying patches?

Reply:
This fork has downgraded a bunch of packages to make it work with Rust 1.73. I don't think the author wants that on the main repo.

Reply:
Is the problem that we use an old (as in less than 6-month old) version of Rust?  We can update that, if that helps.

Reply:
Yes, that should fix it! :) I did consider trying to make a PR to BB, but it does seem excessive to have to keep up with Rust's many releases.
Nonetheless, if you have time to either upgrade Rust, or teach me how to, that'd be great!

Reply:
@giordano If you have time to upgrade the Rust toolchain to the latest version that's great, else having this functional patch out is the second best.

Reply:
We have rustc 1.77.1 now.

Reply:
If only Rust projects weren't aggressively requiring versions of rust more recent than 6 months ago... :smiling_face_with_tear: